template/src/transformPreprocessor.js

```diff
@@ -0,0 +1,15 @@
+const babelJest = require('babel-jest');
+
+// Get the babelConfig
+// This one should be somehow optimized
+const preactCLIBabelRC = require('preact-cli/lib/lib/babel-config.js');
```

Comment:
Agreed - there's a PR by @zouhir that makes this beautiful:
https://github.com/developit/preact-cli/pull/405

Reply:
Was going to mention this too. That should get in soon, perhaps in next patch.

Reply:
Oh we definitely should get that in.

Reply:
Yeah it's awesome that @zouhir 's PR is merged, once a patch is released I can definitely updated that line

Reply:
It's now out as `2.1.0`

Reply:
Great! Let me then update accordingly before you merge ?